modules/programs/convert_img/tests/test_convert_img.py

```
@@ -0,0 +1,507 @@
+"""
+Comprehensive tests for convert_img module.
+"""
+import base64
+import subprocess
```

Ruff (0.11.9)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)